sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2023,11 +2041,29 @@ object functions {
 
   /**
    * @param columnName hyperbolic angle
-   * @return hyperbolic sine of the given value, as if computed by `java.lang.Math.sinh`
+   * @return inverse hyperbolic sine of the given value
    *
    * @group math_funcs
-   * @since 1.4.0
+   * @since 3.0.0
    */
+  def asinh(columnName: String): Column = asinh(Column(columnName))
```

Comment:
Shall we remove those APIs in Scala and R sides?

Reply:
They don't exist already, right?

Reply:
Hi @HyukjinKwon, do you mean we remove then from R for the same reason we removed them from Python, and just keep them in Scala?

Reply:
Sorry I had to clarify it. Yes, let's add it only into SQL side and remove them in Scala, Python and R per https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L42-L56

These are only added into other language APIs when they are commonly used. I think we don't have to add them in the first place in general.

Reply:
thanks Hyukjin. just removed them from language APIs. now only available in SQL